src/pkg/projects/netcoreapp/sfx/Microsoft.NETCore.App.SharedFx.sfxproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <SfxIdentity>Microsoft.NETCore.App</SfxIdentity>
+    <SfxPackageIdentity>$(SfxIdentity).Internal</SfxPackageIdentity>
+
+    <!-- Lay out the sfx in the legacy location for old packaging to keep working for now. -->
```

Comment:
More changes are coming?

Reply:
Not as part of this migration. This is an old comment:

https://github.com/dotnet/core-setup/blob/6a78e0fd349ef19a5a31380c37b39e62ad68fefa/src/pkg/projects/netcoreapp/pkg/workaround/Microsoft.NETCore.App.pkgproj#L28-L29

Updating to `Lay out the sfx in the legacy location for old packaging to keep working for now. This can be removed when the legacy infra is phased out.` to be a little more direct.